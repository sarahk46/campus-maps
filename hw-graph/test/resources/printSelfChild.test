# Tests that a graph along with a node and an edge to itself can be created.
# Further tests if such edge is printed.

# Create a graph
CreateGraph graph1

# Add a node
AddNode graph1 n1

# Add an edge
AddEdge graph1 n1 n1 e1

# Print the outgoing edges from n1
ListChildren graph1 n1