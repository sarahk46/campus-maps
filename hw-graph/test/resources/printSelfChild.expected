# Tests that a graph along with a node and an edge to itself can be created.
# Further tests if such edge is printed.

# Create a graph
created graph graph1

# Add a node
added node n1 to graph1

# Add an edge
added edge e1 from n1 to n1 in graph1

# Print the outgoing edges from n1
the children of n1 in graph1 are: n1(e1)